src/release.js

```diff
@@ -613,7 +613,15 @@ export default function taoExtensionReleaseFactory(params = {}) {
         async removeReleasingBranch() {
             log.doing('Clean up the place');
 
-            await gitClient.deleteBranch(data.releasingBranch);
+            try {
+                await gitClient.deleteBranch(data.releasingBranch);   
```

Comment:
I'm not sure if attempting to delete the branch or checking the existence of the branch before deleting but it resolves the issue either way :) 

Bom trabalho!

Reply:
Thanks!

Yes, I thought the same, but it can save one git request if we try to delete it directly :)

Reply:
agreed!